flink-connectors/flink-connector-kinesis/src/test/java/org/apache/flink/streaming/connectors/kinesis/FlinkKinesisConsumerMigrationTest.java

```diff
@@ -1,149 +0,0 @@
-/*
```

Comment:
I discussed with @aljoscha that this class can be resurrected when merging #4565